api/python/flaschen.py

```diff
@@ -34,15 +56,15 @@ def __init__(self, host, port, width, height, layer=5, transparent=False):
     self.transparent = transparent
     self._sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
     self._sock.connect((host, port))
-    header = ''.join(["P6\n",
-                      "%d %d\n" % (self.width, self.height),
-                      "255\n"]).encode('utf-8')
-    footer = ''.join(["0\n",
-                      "0\n",
-                      "%d\n" % self.layer]).encode('utf-8')
-    self._data = bytearray(width * height * 3 + len(header) + len(footer))
+    header = _HEADER_P6_FT % {
+      b"width": self.width,
+      b"height": self.height,
+      b"x": 0,
+      b"y": 0,
+      b"z": self.layer,
+    }
+    self._data = bytearray(width * height * 3 + len(header))
```

Comment:
I'd probably write `len(header) + width * height * 3` so that it better matches the mental model of the layout of the buffer, so makes it easier on someone reading the code.

Reply:
Done.